app/tests/spec/views/complete_reset_password.js

```diff
@@ -273,7 +271,7 @@ function (chai, sinon, p, AuthErrors, Metrics, FxaClient, InterTabChannel,
       });
 
       it('non-direct-access signs the user in and redirects to `/reset_password_complete` if broker does not say halt', function () {
-        var account;
+        var account; //eslint-disable-line no-unused-vars
```

Comment:
Changing the assertions to this results in the following assertionerrors being thrown:
![screen shot 2015-06-12 at 1 11 03 pm](https://cloud.githubusercontent.com/assets/432707/8138962/9c2b87fa-1104-11e5-94ea-e89a96d20f69.png)
@zaach